akka-stream/src/main/mima-filters/2.5.7.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# #23808 adding simplified throttle API
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.stream.scaladsl.FlowOps.relaxedThrottle")
```

Comment:
It should be in 2.5.6 file since the newly introduced incompatibility needs to be filtered up until 2.5.6.

Reply:
got it.

Reply:
Do we have this documented somewhere? I remember wondering where to put excludes in the past :)

Reply:
it's documented in CONTRIBUTING.md